src/Core/System/SystemConfig/SystemConfigService.php

```diff
@@ -79,7 +81,10 @@ public function get(string $key, ?string $salesChannelId = null)
 
         foreach ($parts as $part) {
             if (!\is_array($pointer)) {
-                return null;
+                $event = new SystemConfigLoadedEvent($key, null, $salesChannelId);
```

Comment:
not 100% sure if we need it for the null values (missing values), for now I added it but I can remove those again.